packer/files/backend-config.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Since this is a template and not actually true JSON can we change the filename to reflect that

`backend-config.json.template` or something like that?

Reply:
If we wanted to make this change it would affect more than just this file, so if its something we want to do, we should do it in a separate PR